examples/subcurrency/src/main.sw

```diff
@@ -1,11 +1,15 @@
 // ANCHOR: body
 contract;
 
-use std::chain::*;
-use std::hash::*;
-use std::panic::panic;
-use std::storage::*;
-use std::address::Address;
+use std::{
+    address::Address, 
+    assert::assert, 
+    chain::auth::{AuthError, Sender, msg_sender}, 
+    hash::*, 
+    panic::panic, 
+    result::*, 
+    storage::{get, store}
+};
```

Comment:
I did cheat here and format this `use` statement manually. Relevant issue: https://github.com/FuelLabs/sway/issues/1209